rrrspec-client/lib/rrrspec/client/slave_runner.rb

```diff
@@ -17,7 +17,6 @@ def initialize(slave, working_dir, taskset_key)
         @slave = slave
         @taskset = Taskset.new(taskset_key)
         @timeout = TASKQUEUE_TASK_TIMEOUT
-        @rspec_runner = RSpecRunner.new
         @working_path = File.join(working_dir, @taskset.rsync_name)
```

Comment:
As noted above, I don't want to put an RSpecRunner object in a global state. I want to make runner instances local to SlaveRunner.

In general, using global variables is not good design. It's better to make the scope narrow as much as possible. Plus, if you need to have some object in a wider scope, it's better to make the object owner clear in order to control the object lifecycle.